jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/AbstractCoder.java

```diff
@@ -86,18 +88,21 @@ public void decodeFromStreamAndPublish(InputStream is, String topic) throws IOEx
    }
 
    @Override
-   public void publish(String topic, String msg) {
+   public void publishJson(String msg) {
+       System.out.println("alpha");
         MessageProducer
                 .defaultStringMessageProducer(odeProperties.getKafkaBrokers(), odeProperties.getKafkaProducerType())
-                .send(topic, null, msg);
+                .send(OdeProperties.KAFKA_TOPIC_J2735_BSM_JSON, null, msg);
+        
+        System.out.println("beta");
 
         logger.debug("Published: {}", msg);
     }
 
    @Override
-    public void publish(String topic, byte[] msg) {
+    public void publishByte(byte[] msg) {
```

Comment:
no need to rename these methods. Overloading will resolve the methods. But if you want to keep the new names, this should be called publoshByteArray or publishBytes.